containers_test.go

```diff
@@ -201,3 +202,78 @@ func TestUpload(t *testing.T) {
 		t.Errorf("%s not found", path)
 	}
 }
+
+func TestSquashImage(t *testing.T) {
+	client := DockerClient()
+	if client == nil {
+		t.Skip("Could not find Docker daemon connection")
+	}
+
+	repo := "yourbase-layer-test"
+	tag := "v1"
+	image := fmt.Sprintf("%s:%s", repo, tag)
```

Comment:
nit: You can make these into compile-time constants.

```suggestion
	const (
		repo = "yourbase-layer-test"
		tag = "v1"
		image = repo + ":" + tag
	)
```